src/tests/ftest/util/job_manager_utils.py

```diff
@@ -165,6 +165,19 @@ def _get_remote_process_state(self, message=None):
         # processes running by returning a "R" state.
         return "R" if 1 not in results or len(results) > 1 else None
 
+    def kill(self):
+        """Forcibly terminate any job processes running on hosts."""
+        regex = self.job.command_regex
+        result = stop_processes(self._hosts, regex)
+        if 0 in result and len(result) == 1:
+            self.log.info(
+                "No remote {} processes killed (none found), done.".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments

Reply:
Fixed